### PR TITLE
How to guides

### LINKED ISSUE #42: Write vignettes
Articles:

1. What `box.linters::box_default_linters` cover.
2. What `box.linters::rhino_default_linters` cover.
3. How to customize a configuration.

### PR DESCRIPTION
**Pull request by radbasa**
*Have you read the [Contributing Guidelines](https://github.com/Appsilon/.github/blob/main/CONTRIBUTING.md)?*

Closes #42 

## Description
Three simple articles:
1. custom lintr configuration
2. running lintr with a project
3. running lintr with a rhino project

## Definition of Done
- [ ] The change is thoroughly documented.
- [ ] The CI passes (`R CMD check`, linter, unit tests, spelling).
- [ ] Any generated files have been updated (e.g. `.Rd` files with `roxygen2::roxygenise()`)

### CODE REVIEW

File: pkgdown/_pkgdown.yml [hunk 1]
```diff
@@ -38,10 +38,14 @@ navbar:
       text: "Getting Started"
       href: index.html
     tutorials:
-      text: Tutorials
+      text: How-to Guides
       menu:
         - text: Custom configuration
-          href: articles/tutorial/custom-config.html
+          href: articles/how-to/custom-config.html
```
Comment (jakubnowicki) [inline pkgdown/_pkgdown.yml:44]: Custom configuration should be the last article here.

File: vignettes/how-to/rhino-default.Rmd [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Projects created with `rhino` version 1.8.0 and later come preconfigured with a `.lintr` file. 
+
+```{yaml}
+linters:
+  linters_with_defaults(
+    defaults = box.linters::rhino_default_linters,
+    line_length_linter = lintr::line_length_linter(100)
+  )
+encoding: "UTF-8"
```
Comment (jakubnowicki) [inline vignettes/how-to/rhino-default.Rmd:27]: As Rhino provides the Rproj file, `lintr` uses the encoding that is available there. Please remove it from here.

File: pkgdown/_pkgdown.yml [hunk 3]
```diff
@@ -38,10 +38,14 @@ navbar:
       text: "Getting Started"
       href: index.html
     tutorials:
```
Comment (jakubnowicki) [inline pkgdown/_pkgdown.yml:40]:
```suggestion
    how-to-guides:
```

File: vignettes/how-to/rhino-default.Rmd [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Rhino (>= 1.8.0)
```
Comment (jakubnowicki) [inline vignettes/how-to/rhino-default.Rmd:17]: Please add a note, that Rhino 1.8 is not yet available (and an issue to remove the note after the Rhino release).

**Review by jakubnowicki — APPROVED** [review]
Looks good. Please address my comments and you can merge.

File: vignettes/how-to/rhino-default.Rmd (hunk 4 above)
Comment (radbasa) [reply vignettes/how-to/rhino-default.Rmd:17]: Yeah, I was going to ask about version sync with rhino.

File: pkgdown/_pkgdown.yml (hunk 1 above)
Comment (radbasa) [reply pkgdown/_pkgdown.yml:44]: Done

File: pkgdown/_pkgdown.yml (hunk 3 above)
Comment (radbasa) [reply pkgdown/_pkgdown.yml:40]: Done. Also forgot the label in structure.

**Pull request merged**